volatility/framework/layers/crash.py

```diff
@@ -146,7 +146,7 @@ def _load_segments(self) -> None:
                 offset += x.PageCount
 
         elif self.dump_type == 0x05:
-            summary_header.BufferLong.count = (summary_header.BitmapSize + 31) // 32
+            summary_header.BufferLong.count = (summary_header.BitmapSize + 31) // 32 + 0x2000
```

Comment:
Could you document why you're making this change please?  Why `0x2000` in particular?  Is this the offset for the first two pages?